xml/depl_maintenance.xml

```diff
@@ -1503,6 +1503,304 @@ migrate</screen>
     the selected nodes.
    </para>
   </sect2>
+  <sect2 xml:id="sec.depl.maintenance.upgrade.troubleshooting">
+   <title>Common Upgrade Problems and Possible Solutions</title>
+    <qandaset>
+     <qandaentry>
+      <question>
+       <para>
+	Upgrade of the admin server has failed.
+       </para>
+      </question>
+      <answer>
+       <para>
+	Check for empty, broken, and not signed repositories in the &admserv;
+	upgrade log file <filename>/var/log/crowbar/admin-server-upgrade.log</filename>. Fix the
```

Comment:
"... in the &admserv; and upgrade log file..

Reply:
I think the original version is correct. Basically it means that you have to check the upgrade log file on the admin server. @jsuchome Correct?

Reply:
Yes, the /var/log/crowbar/admin-server-upgrade.log log file is located on admin server

Reply:
Ah I see it now :)